samples/servicecontrol/adapter-rabbitmq-different-topologies/sample.md

```diff
@@ -10,7 +10,7 @@ related:
 ---
 
 
-This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using the RabbitMQ direct routing topology. The direct routing topology is used by endpoints that are using NServiceBus Version 6. ServiceControl is built on NServiceBus Version 5 and can only use the conventional routing topology. The ServiceControl transport adapter is required to bridge both routing topologies.
+This sample shows how to configure ServiceControl to monitor endpoints and retry messages when using the RabbitMQ direct routing topology, the default topology of endpoints using RabbitMQ on NServiceBus Version 6. However, since ServiceControl is built on NServiceBus Version 5 and can only use the conventional routing topology, a kind of "transport adapter" is required to bridge the routing topologies. 
```

Comment:
@mauroservienti The direct routing topology is **not** the default for any version of the RabbitMQ transport. 

The scenario being tested here is when someone has chosen to use the direct routing topology instead of the default because there is no way to configure ServiceControl to use something other than the default. It has nothing to do with ServiceControl being built on NSB 5, either. The direct routing topology is also available on NSB 5. The problem is that there's no way to configure transport settings inside of ServiceControl.